modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -12,67 +12,66 @@ endif::[]
 :meta-product: API Manager, Studio, Mule
 :meta-applies-to:
 
-Mule OAuth 2.0 Provider is an OAuth 2.0 provider alternative developed by MuleSoft that can be used in any API Platform organization (including the Federated ones).
-It is the recommended initial solution pending future corporate investments in enterprise specific OAuth 2.0 offerings.
+The Mule OAuth 2.0 Provider is an alternative from MuleSoft for the OAuth 2.0 provider, and can be used in any API Platform organization, including the Federated ones. The Mule OAuth 2.0 provider is the first-step solution for your OAuth 2.0 offerings.
 
-Supported third party OAuth 2.0 providers:
+You must deploy the Mule OAuth 2.0 provider to a runtime with API gateway capabilities, for example, Mule runtime engine (Mule) versions 3.x, 4.2.0, or later, with the corresponding organization credentials. The following third party OAuth 2.0 providers are supported:
 
 * PingFederate
 * OpenAM
 * OAuth 2.0 providers supporting Open ID Connect
 
-The xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] is designed to work with the Mule OAuth 2.0 provider.
+The xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 access token enforcement policy] works with the Mule OAuth 2.0 provider.
```

Comment:
Changes the meaning of the statement. Works seem to imply that Mule OAuth 2.0 access token enforcement policy is one of several policies that can work with this, while we are stating that the Mule OAuth provider was designed with that particular policy in mind, and only that one.

Reply:
changed reverted! Kim please verify and suggest the best alternative.